ext/mochilo/mochilo_api.h

```diff
@@ -4,6 +4,30 @@ MOAPI mo_value moapi_bytes_new(const char *src, size_t len)
 	return (mo_value)rb_str_new(src, len);
 }
 
+MOAPI mo_value moapi_symbol_new(const char *src, size_t len)
+{
+	return ID2SYM(rb_intern(src));
```

Comment:
Yes! I am going to use that instead. I didn't think to look for `rb_intern2`, but it's exactly what I wanted to use here.